package.json

```diff
@@ -39,6 +41,7 @@
 		"lint-staged": "^7.0.0",
 		"lolex": "^2.1.3",
 		"prettier": "1.14.2",
+		"prom-client": "^11.1.2",
```

Comment:
Love testing against a published version. 👍 

This might be a non-issue, but we'll have to update this (and the lock files) after every release (or at least when we care about benchmarks). I don't see a way to exclude a module from the lock files, which would give some options for improving that situation. Not listing it as a dependency at all could be another option, and then we could put the `require` for it in a try/catch?

Reply:
we could add a CI check or something? `npm show prom-client version` returns latest published, and if the local version does not match, throw?

Reply:
Yeah, that is interesting. We could have a `postinstall` step that only runs locally that installs the latest version? But yeah, I hadn't thought of the lockfile issue. Let me know what you prefer.

Reply:
Oo, using a package.json script is another good idea. Wouldn't want it to run all the time though, so if it's in `postinstall`, would have to check somehow if it's being installed in a git checkout or as a dependency, I think?

(Might not be worth the trouble to make this "nice" -- could just manually bump it as necessary.)